baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/PreferSafeLoggerTest.java

```diff
@@ -210,4 +210,48 @@ void testPassedLoggerUnmodified() {
                 .expectUnchanged()
                 .doTest();
     }
+
+    @Test
+    void testGetNameInArgUnmodified() {
+        // getName is not supported by SafeLogger, so we shouldn't make changes that won't compile.
+        fix().addInputLines(
+                        "Test.java",
+                        "import com.palantir.logsafe.*;",
+                        "import org.slf4j.*;",
+                        "class Test {",
+                        "  private static final Logger log = LoggerFactory.getLogger(Test.class);",
+                        "  void action() {",
+                        "    log.info(\"foo\", SafeArg.of(\"name\", log.getName()));",
```

Comment:
Previously the `log.getName()` method invocation hid in the shadow of the parent log.info node, so the recommended fix wouldn't compile.